lib/import/colorectal/providers/st_george/st_george_handler_colorectal.rb

```diff
@@ -0,0 +1,303 @@
+require 'date'
+
+module Import
+  module Colorectal
+    module Providers
+      module StGeorge
+        # Process St George-specific record details into generalized internal genotype format
+        class StGeorgeHandlerColorectal < Import::Germline::ProviderHandler
+          include Import::Helpers::Colorectal::Providers::Rj7::Constants
+
+          def process_fields(record)
+            genotype = Import::Colorectal::Core::Genocolorectal.new(record)
+
+            # records using new importer should only have SRIs starting with V
+            return unless record.raw_fields['servicereportidentifier'].start_with?('V')
```

Comment:
return statement should be the first statement as we need not to create a genotype object if file is not being processed.

Reply:
changed in [d16287a](https://github.com/NHSDigital/data_management_system/pull/81/commits/d16287ae4da6bcd9dea21fce634bc34266d362ddl)